<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>851</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Transaction</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>851</width>
        <height>471</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Ajouter Transaction</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>761</width>
          <height>311</height>
         </rect>
        </property>
        <property name="title">
         <string>Gestion de Caisse</string>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>111</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Nom Du Mr/Mme</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>131</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Num Du Client </string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>40</y>
           <width>181</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>70</y>
           <width>181</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>250</y>
           <width>171</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>131</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Date De Transaction</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_5">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>140</y>
           <width>181</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>140</y>
           <width>131</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Prix Totale</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>170</y>
           <width>131</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Adresse Du Mr/Mme </string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>210</y>
           <width>131</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Id Utilisateur </string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_7">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>210</y>
           <width>181</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QDateTimeEdit" name="dateTimeEdit">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>100</y>
           <width>181</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>180</y>
           <width>181</width>
           <height>22</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>Tunis</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ariana</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>L'Aouina </string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Marsa</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Gamart</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Soukra </string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Carthage </string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sidi BouSaid </string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Manazah</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hay Nasr</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Bardo</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Rades</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Megrine</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mourouj City</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Naasen </string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Fouchena</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Nabeul</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hammamet</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sousse</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sfax</string>
          </property>
         </item>
        </widget>
       </widget>
       <widget class="QLineEdit" name="textEdit_2">
        <property name="geometry">
         <rect>
          <x>512</x>
          <y>340</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>350</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>ID_Transaction</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>340</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Consulter Liste Des Transaction</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>360</y>
          <width>111</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="textDelete_2">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>360</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableView" name="tableView_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>90</y>
          <width>721</width>
          <height>211</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>360</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>710</x>
          <y>60</y>
          <width>80</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>TRIER </string>
        </property>
       </widget>
       <widget class="QPushButton" name="rechercher">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>20</y>
          <width>80</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Rechercher</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>20</y>
          <width>411</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Evaluation</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>861</width>
        <height>511</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Ajouter Une Evaluation</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>10</y>
          <width>631</width>
          <height>351</height>
         </rect>
        </property>
        <property name="title">
         <string>Evaluation</string>
        </property>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>111</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Nom Du Mr/Mme</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>131</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Note Client </string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>40</y>
           <width>181</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>424</x>
           <y>280</y>
           <width>191</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>131</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Remarque Du Client </string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_9">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>140</y>
           <width>181</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>140</y>
           <width>131</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Id Facture</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_2">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>70</y>
           <width>181</width>
           <height>22</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>11</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>13</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>14</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>15</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>17</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>18</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20</string>
          </property>
         </item>
        </widget>
        <widget class="QComboBox" name="comboBox_3">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>110</y>
           <width>181</width>
           <height>22</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>Fantastique </string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Bonne </string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Moyenne </string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mediocre</string>
          </property>
         </item>
        </widget>
       </widget>
       <widget class="QLineEdit" name="textEdit_4">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>370</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_8">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>370</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>370</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Consulter Les Evaluations</string>
       </attribute>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>791</width>
          <height>201</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>340</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="textDelete">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>340</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>340</y>
          <width>171</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="rechercher_2">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>10</y>
          <width>80</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Rechercher</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>10</y>
          <width>411</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>847</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
